src/content/blog/2024-08-27-integration-tests.mdx

```diff
@@ -0,0 +1,130 @@
+---
+layout: ../../layouts/BlogLayout.astro
+title: "Leveling Up Rafiki Testing: Shifting from Manual to Automated"
+description: "How we automated our manual payment flow tests."
+date: 2024-08-27
```

Comment:
Can you update this to be next week Tuesday?

Reply:
Well that Tuesday has come and gone... I changed to 9/10 but not sure what the planned date is now... can revise to whatever we need.

Reply:
changed to 2024-09-06